app/controllers/admin/users_controller.rb

```diff
@@ -4,26 +4,28 @@ class Admin::UsersController < ApplicationController
   def show
     user = User.find(params[:id])
     authorize user
-    render :show, locals: { user: user}
+    render :show, locals: { user: user }
   end
 
   def index
-    q = User.ransack(params[:q])
+    q = User.from_current_user_organization(current_user.organization_id).ransack(params[:q])
```

Comment:
rename the first scope to: `User.in_organization(current_organization)` - pass the organization object. Btw add a method `current_organization` which will hold that current organization